addons/mail/controllers/discuss/rtc.py

```diff
@@ -131,10 +131,13 @@ def channel_ping(self, channel_id, rtc_session_id=None, check_rtc_session_ids=No
             channel_member_sudo.channel_id.rtc_session_ids.filtered_domain(domain).write({})  # update write_date
         current_rtc_sessions, outdated_rtc_sessions = channel_member_sudo._rtc_sync_sessions(check_rtc_session_ids)
         return (
-            Store(member.channel_id, {"rtcSessions": Store.many(current_rtc_sessions, "ADD")})
+            Store(
+                member.channel_id,
+                Store.Value("rtcSessions", Store.Many(current_rtc_sessions, mode="ADD")),
```

Comment:
```suggestion
               Store.Many(current_rtc_sessions, mode="ADD", rename="rtcSessions")
```

Reply:
as discussed, not possible because the rename is done in a different pass (only when field name is provided, not records)